addons/pandora/plugin.gd

```diff
@@ -15,6 +15,8 @@ func _init() -> void:
 func _enter_tree() -> void:
 	add_autoload_singleton("Pandora", "res://addons/pandora/api.gd")
 	
+	PandoraSettings.initialize()
```

Comment:
Could we make these settings part of the project settings instead? https://docs.godotengine.org/en/stable/classes/class_projectsettings.html